### ISSUE TITLE
Use without bevy

### ISSUE THREAD

**Issue by tversteeg** [Leafwing-Studios/leafwing_manifest#39]
Would it be possible to make the integration with bevy an optional feature? I would love to use this in my own game engine, and I think the Rust ecosystem as a whole can benefit from a solution like this.

**Comment by alice-i-cecile** [comment]
Yeah, I was wondering if others would find this useful :) https://github.com/Leafwing-Studios/leafwing_manifest/pull/21 was an attempt at this: if you'd like, I'd be happy to review a PR that revived it!
